submissions/zhenyakornilov/js-dom/js/index.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+
+import museums from "./data.js";
+import switchTheme from "./utils.js";
+
+function main() {
+  const dropdownBtn = document.getElementById("dropdown");
+  const listMenu = document.getElementById("menu");
+  const contentBody = document.getElementById("content");
+  renderButtons(museums, listMenu);
+  renderArticle(museums[0], contentBody);
+  showMuseumInfoByClick(museums, contentBody, listMenu);
+  openMenu(listMenu, dropdownBtn);
+  switchTheme();
+}
+
+function renderButtons(museumsObj, menu) {
+  let buttonsHTML = museumsObj
+    .map(({ btnName }) => {
+      return `
+      <li class="nav-item">
+        <button class="btn-text">${btnName}</button>
+      </li>
+      `;
+    })
+    .join("");
+
+  menu.innerHTML = buttonsHTML;
+}
+
+function showMuseumInfoByClick(museums, content, menu) {
+  menu.addEventListener("click", function ({ target }) {
+    if (target.tagName.toLowerCase() === "button") {
+      const currentContentObject = museums.filter(
+        ({ btnName }) => btnName === target.textContent
+      );
+      renderArticle(currentContentObject[0], content);
+    }
+  });
+}
+
+function renderArticle({ name, description, imageSrc, url }, content) {
+  const contentHTML = `
+  <h2 class="museum-title">${name}</h2>
+  <div class="desc-wrapper">
+    <img class="museum-image" src="${imageSrc}" alt="photo of museum">
```

Comment:
`alt` should be moved to `data.js` as one of the paramters, because it can't be the same for different images